todos/views.py

```diff
@@ -32,4 +36,16 @@ def update(request, todo_id):
     todo.isCompleted = isCompleted
 
     todo.save()
-    return redirect('todos:index')
\ No newline at end of file
+    return redirect('todos:index')
+
+class RegisterView(generic.CreateView):
+    form_class = UserCreationForm
+    template_name = 'registration/register.html'
+    success_url = reverse_lazy('login')
+
+    def form_valid(self, form):
+        valid = super(RegisterView, self).form_valid(form)
+        username = form.cleaned_data.get('username')
```

Comment:
Where is the minimum check length of the password. We want the password length to be of 6 characters atleast